tools/depends/configure.ac

```diff
@@ -263,17 +263,6 @@ case $build in
 esac
 
 AC_PROG_CPP
-AX_CXX_COMPILE_STDCXX_14([noext],[mandatory])
-
-case $build in
```

Comment:
The removal here looks incorrect. This is done to remove flags used for compiler tests, otherwise Mac has chance of failing cpp test with no sysroot. However, those flags are host, not platform, so we remove them after ac tests are complete. Sysroot is essentially considered always required for proper apple builds

Reply:
Have just confirmed returning this block will get the ios/tvos configure to build correct